### PR TITLE
CondCore - remove newly committed odr violation; fix warning from boost

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: CondCore/CondDB/plugins/RelationalAuthenticationService.cc [hunk 1]
```diff
@@ -16,15 +16,17 @@
 #include <memory>
 #include <sys/stat.h>
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```
Comment (Dr15Jones) [inline CondCore/CondDB/plugins/RelationalAuthenticationService.cc:19]: The `std::` has a bind and a `std::function` as well.

**Comment by ggovi** [comment]
+1

File: CondCore/CondDB/plugins/RelationalAuthenticationService.cc (hunk 1 above)
Comment (qliphy) [reply CondCore/CondDB/plugins/RelationalAuthenticationService.cc:19]: @Dr15Jones  Do you have further comment on this PR?

Comment (davidlange6) [reply CondCore/CondDB/plugins/RelationalAuthenticationService.cc:19]: @Dr15Jones is correct. We did a campaign a year ago but it didn't cover this one. I'm just fixing a warning in this PR, but would support a migration by @l2_db

**Comment by qliphy** [comment]
+1

**Pull request merged**